package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "twilio-live-interactive-audio",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
There's a version number in the package-lock file too, not that it makes a huge difference.